src/tools/lib/python/picongpu/utils/find_time.py

```diff
@@ -0,0 +1,166 @@
+"""
+This file is part of the PIConGPU.
+
+Copyright 2017 PIConGPU contributors
+Authors: Axel Huebl
+License: GPLv3+
+"""
+
+import numpy as np
+import os
+
+
+class FindTime(object):
+    """
+    Convert iterations (time steps) to time [seconds] and back.
+    """
+
+    def __init__(self, run_directory):
+        """
+        Parameters
+        ----------
+        simulation_directory : string
+            path to the run directory of PIConGPU
+            (the path before ``simOutput/``)
+        """
+        if run_directory is None:
+            raise ValueError('The run_directory parameter can not be None!')
+
+        self.run_directory = run_directory
+        self.data_file = "output"
```

Comment:
note: relying on this file is still a bit messy, especially during restarts, but we can generate a cleaner log & lookup of basic parameters later on, too